pac4j-cas/src/test/java/org/pac4j/cas/client/rest/TestCasRestClient.java

```diff
@@ -0,0 +1,27 @@
+package org.pac4j.cas.client.rest;
+
+import org.pac4j.core.profile.CommonProfile;
+
+/**
+ * The {@link TestCasRestClient} is responsible for
+ * executing tests against {@link CasRestFormClient}.
+ *
+ * @author Misagh Moayyed
+ */
+public class TestCasRestClient extends org.pac4j.core.profile.TestProfileHelper  {
```

Comment:
I messed up things around tests naming. Just to let you know that unit tests can be named TestSomething, SomethingTest or SomethingTests. My fault.